addons/web_editor/static/src/xml/editor.xml

```diff
@@ -268,6 +268,24 @@
         </we-customizeblock-options>
     </t>
 
+    <!-- Collaboration reset dialog -->
+    <t t-name="web_editor.collaboration-reset-dialog">
+        <div>
+            <div style="color: red;">
+                <p>
+                    There is a conflict between your version and the one in the database.
```

Comment:
@dmo-odoo @Goaman Should not have this been added in the .pot file so it can be translated as merged in stable?